website/events/api/v2/filters.py

```diff
@@ -0,0 +1,82 @@
+from django.utils import timezone
+from rest_framework import filters
+
+from utils.snippets import extract_date_range
+
+
+class EventDateFilterBackend(filters.BaseFilterBackend):
+    """Allows you to filter by event start/end dates."""
+
+    def filter_queryset(self, request, queryset, view):
+        start, end = extract_date_range(request, allow_empty=True)
+
+        if start is not None:
+            queryset = queryset.filter(end__gte=start)
```

Comment:
doesn't `start_gte` make more sense?

Reply:
Nope, that would exclude any current events. See https://github.com/svthalia/concrexit/pull/1592#discussion_r604375090

Reply:
I see, I thought it was the opposite hahaha. Always difficult those calculations with time ranges